tcp_queue_length/manifest.json

```diff
@@ -2,7 +2,7 @@
   "maintainer": "help@datadoghq.com",
   "manifest_version": "1.0.0",
   "metric_prefix": "tcp_queue.",
-  "metric_to_check": "tcp_queue.rqueue.size",
+  "metric_to_check": "tcp_queue.read_buffer_max_fill_rate",
```

Comment:
Is the previous metric not available in the newer versions of the check?

Reply:
Yes indeed. There was a cardinality issue with the previous metric.